docs/develop/update/firewall.rst

```diff
@@ -65,3 +69,116 @@ When making changes to firewall settings or port assignments, you might want to:
    .. code-block:: bash
 
       netstat -tupln
+
+Docker Servers
+==============
+
+Docker manipulates IPTables to route network traffic to and from containers this interferes with our firewall management script. To prevent errors saltstack and the firewall script identify docker and block the script from running. To work around this on servers hosting Docker we need to implement an alternative firewall solution that is configured external to the instance.
+
+Hetzner (Hardware Servers)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Hetzner provide a simple stateless firewall with each server. "Stateless" means that the firewall does not track connections, it simply monitors all inbound and outbound traffic by IP and ports in that moment.
+
+You can configure the Hetzner firewall as follows:
+
+#. `Log into Hetzner <https://robot.your-server.de/server>`__
+#. Select your server and go to the *Firewall* tab
+#. Set *Status* to active
+#. Enable *Hetzner Services*
+#. Create your firewall rules
+
+   We recommend the following rules as a minimum
+
+   .. list-table::
+       :header-rows: 1
+
+       * - Name
+         - Source IP
+         - Destination IP
+         - Source port
+         - Destination port
+         - Protocol
+         - TCP flags
+         - Action
+       * - Allow SSH
+         - 0.0.0.0/0
+         - 0.0.0.0/0
+         - 0-65535
+         - 22
+         - *
+         -
+         - Accept
+       * - Allow ICMP
+         - 0.0.0.0/0
+         - 0.0.0.0/0
+         - 0-65535
+         - 0-65535
+         - icmp
+         -
+         - Accept
+       * - Allow Prometheus
+         - 213.138.113.219/32
+         - 0.0.0.0/0
+         - 0-65535
+         - 7231
+         - *
+         -
+         - Accept
+       * - Allow Outgoing TCP
+         - 0.0.0.0/0
+         - 0.0.0.0/0
+         - 0-65535
+         - 32768-65535
+         - tcp
+         - ack
+         - Accept
+
+.. Note:::
+
+   `More information can be found in the Hetzner documentation.<https://docs.hetzner.com/robot/dedicated-server/firewall/>`__
+
+Linode (VPS Servers)
+~~~~~~~~~~~~~~~~~~~~
+
+Linode provide a stateful Cloud Firewall. Stateful firewalls track connections for you making configuration easier.
```

Comment:
```suggestion
Linode provide a stateful `Cloud Firewall <https://www.linode.com/docs/guides/getting-started-with-cloud-firewall/>`__. Stateful firewalls can store information about connections across time, which is required for use cases like port knocking.
```

Reply:
Thinking of rephrasing to "store information abut connections over time" as this is the more common phrasing.

Happy with "across time" too if it was chosen for a specific reason?

Reply:
"over" is also fine with me.